modules/odc-using-the-devfile-registry.adoc

```diff
@@ -0,0 +1,15 @@
+:_content-type: PROCEDURE
+[id="odc-using-the-devfile-registry_{context}"]
+= Using the Devfile Registry to access product Devfiles
+
+You can use the Devfiles in the  in the *+Add* flow of the *Developer* perspective to create an application. The Devfile registry provides a complete integration with the public community registry and the product registry. A devfile is a portable YAML file that describes your development environment without needing to configure it from scratch. Using the Devfile registry, you can use a pre-configured devfile to create an application.
+
+.Procedure
+
+. In the *Developer perspective*, navigate to the *+Add* view and from the Developer Catalog tile, click *All Services* to view all the available services in the Developer Catalog.
+
+. Under *All Services*, select Devfiles where you can browse for devfile that support a particular language or framework. Alternatively, you can use the keyword filter to search for a particular devfile using their name, tag or description.
+
+. Click the devfile you want to use to create an application. The devfile tile will display the details of the devfile which includes the name, description, provider and the documentation of the devfile.
```

Comment:
- "will display" => "displays"
- Comma after "devfile"
- "which includes" => "including"
- Comma after "provider" (learn about serial comma in US English)